proof/invariant-abstract/AARCH64/ArchCNodeInv_AI.thy

```diff
@@ -389,10 +389,10 @@ lemma swap_of_caps_valid_arch_caps [CNodeInv_AI_assms]:
      apply (rule conjI; clarsimp)
     apply (intro allI impI)
     apply (elim exE conjE)
-    sorry (* FIXME AARCH64
-    subgoal for _ _ _ _ p
+    subgoal for \<dots> p
```

Comment:
Huh, I don't think that I've seen this before. Does that work backwards like `rename_tac` does?

Reply:
Yes! @Xaphiosis pointed me to that one.